scripts/fish_pixels.py

```diff
@@ -0,0 +1,91 @@
+"""Goes through the fish images for manually marking the ends of the fish"""
+
+import cv2
+import os
+import shutil
+from util import data_path    
+
+file_path = "/Users/georgiamartinez/rov-vision-22/scripts/fish_pixels_list"
```

Comment:
Why not use `data_path` and save the file in the data repo directly instead of making this temporary file and moving it?